pkgs/servers/mail/dovecot/default.nix

```diff
@@ -62,6 +62,7 @@ stdenv.mkDerivation rec {
     "--with-lucene"
     "--with-icu"
   ] ++ lib.optional (stdenv.isLinux) "--with-systemdsystemunitdir=$(out)/etc/systemd/system"
+    ++ lib.optional (stdenv.isDarwin) "--enable-static"
```

Comment:
Please add a comment here or in the commit message explaining why this setting is necessary.